ci/ci

```diff
@@ -0,0 +1,367 @@
+            sys.exit(1)
+        args = ap.parse_args(sys.argv[1:])
+    except Exception as e:
+        log_error(e)
+    if args.pbs_dir != None:
+        pbs_dirname = os.path.join(args.pbs_dir, 'ci')
+        try:
+            os.chdir(pbs_dirname)
+        except Exception as e:
+            log_error(e)
+            log_error("Was correct pbs path provided")
+            sys.exit(1)
+    if args.delete == None and args.start == None and args.local == None:
+        ret = check_prerequisites()
+        if ret == 1:
+            log_error(
+                "container build failed, build logs can be found in " + build_log_path)
+            sys.exit(1)
+
+    try:
+        if args.configure != None:
+            val = args.configure
+            configure_opt = os.path.join(pbs_dirname, '.configure_opt')
+            if val == 'called':
+                val = read_from_file(configure_opt)
+                if val == '':
+                    val = '--prefix=/opt/pbs --enable-ptl'
+            log_info("using configure parameters : " + val)
+            write_to_file(configure_opt, val)
+            cmd = "docker-compose exec pbs-build bash -c \'export ONLY_CONFIGURE=1 && /src/do.sh | tee /logs/build \'"
+            run_docker_cmd(cmd)
+            log_info("config options set")
+        if args.make == 'called':
+            cmd = "docker-compose exec pbs-build bash -c \'export ONLY_REBUILD=1 && /src/do.sh | tee /logs/build \'"
+            run_docker_cmd(cmd)
+            log_info(" done make ")
+        if args.install == 'called':
+            cmd = "docker-compose exec pbs-build bash -c \'export ONLY_INSTALL=1 && /src/do.sh | tee /logs/build \'"
+            run_docker_cmd(cmd)
+            log_info(" done install ")
+        if args.test != None:
+            val = args.test
+            benchpress_opt = os.path.join(pbs_dirname, '.benchpress_opt')
+            if val == 'called':
+                val = read_from_file(benchpress_opt)
+                if val == '':
+                    val = '--tags=smoke'
+            log_info("using test parameters : " + val)
+            write_to_file(benchpress_opt, val)
+            cmd = "docker-compose exec pbs-build bash -c \'export RUN_TESTS=1 && export ONLY_TEST=1 && /src/do.sh | tee /logs/build \'"
+            run_docker_cmd(cmd)
+            log_info(" done test ")
+        if args.pbs_stop == 'called':
+            cmd = "docker-compose exec pbs-build bash -c \' /src/killit.sh  \'"
+            run_docker_cmd(cmd)
+            log_info(" done stop ")
+        if args.run != None:
+            benchpress_opt = os.path.join(pbs_dirname, '.benchpress_opt')
+            val = ''
+            if args.run == 'called':
+                val = read_from_file(benchpress_opt)
+                if val == '':
+                    val = '--tags=smoke'
+            else:
+                val = args.run
+            log_info("using test parameters : " + val)
+            write_to_file(benchpress_opt, val)
```

Comment:
we need not call write_to_file() if read_from_file() returned a non-empty string.